projects/ion/src/lib/pagination/pagination.component.ts

```diff
@@ -35,11 +35,23 @@ export class IonPaginationComponent implements OnChanges, OnInit {
 
   pages: Page[] = [];
 
+  isAdvanced: boolean;
+
+  hoverControl = {
```

Comment:
You could use this variable to manage all states of "more" buttons, if they're visible or hovered.
Something like:

```
moreBtnConfig = {
   left: {hover: false, visible: true}}
   right: {hover: false, visible: true}}
}
```